tests/st/libnetwork/test_mainline_multi_host.py

```diff
@@ -1,136 +0,0 @@
-# Copyright 2015 Tigera, Inc. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-import uuid
-from tests.st.libnetwork.test_mainline_single_host import \
-    ADDITIONAL_DOCKER_OPTIONS, POST_DOCKER_COMMANDS
```

Comment:
We have calico-containers tests that test multi host networking, but none that verify the number of endpoints in the datastore...